projects/ccf-eui/src/app/shared/components/spatial-search-ui-behavior/spatial-search-ui-behavior.component.ts

```diff
@@ -1,21 +1,118 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
-import { MatDialogRef } from '@angular/material/dialog';
-import { SpatialSearchUiComponent } from '../spatial-search-ui/spatial-search-ui.component';
+import { HttpClient } from '@angular/common/http';
+import { ChangeDetectionStrategy, Component, Inject, OnDestroy } from '@angular/core';
+import { MatDialog, MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { SpatialSearch } from 'ccf-database';
+import { InfoDialogComponent, OrganInfo } from 'ccf-shared';
+import { InfoButtonService, PanelData } from 'projects/ccf-shared/src/lib/components/info/info-button/info-button.service';
+import { Subscription } from 'rxjs/internal/Subscription';
+import { SpatialSearchConfigBehaviorComponent } from '../spatial-search-config-behavior/spatial-search-config-behavior.component';
+import { Sex } from '../spatial-search-config/spatial-search-config.component';
+import { CameraSetting, SpatialSearchUiComponent } from '../spatial-search-ui/spatial-search-ui.component';
 
 
+export interface SearchConfigData {
+  sex: Sex;
+  organ: OrganInfo;
+  spatialSearch: SpatialSearch;
+  sliderSettings: number[];
+}
 
 @Component({
   selector: 'ccf-spatial-search-ui-behavior',
   templateUrl: './spatial-search-ui-behavior.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class SpatialSearchUiBehaviorComponent {
+export class SpatialSearchUiBehaviorComponent implements OnDestroy {
+
+  sex: string;
+
+  organ: string;
+
+  spatialSearch: SpatialSearch;
+
+  sliderSettings: number[];
+
+  defaultCamera: CameraSetting;
```

Comment:
Why are you calling these CameraSettings? Let's discuss a better name after standup.